packages/sdk/src/protocols/base_protocol.ts

```diff
@@ -227,10 +222,17 @@ export class BaseProtocolSDK implements IBaseProtocolSDK {
   private async findAdditionalPeers(numPeers: number): Promise<Peer[]> {
     this.log.info(`Finding ${numPeers} additional peers`);
     try {
-      let newPeers = await this.core.allPeers();
+      let newPeers = await this.core.getPeers({
+        maxBootstrapPeers: 0,
+        numPeers: 0
+      });
 
       if (newPeers.length === 0) {
-        this.log.warn("No new peers found.");
+        this.log.warn("No new peers found, trying with bootstrap peers");
+        newPeers = await this.core.getPeers({
+          maxBootstrapPeers: numPeers,
```

Comment:
shouldn't previous line return all bootstrap peers already? 
I mean `0` is for returning all peers, right?